packages/material-ui/src/TablePagination/TablePagination.test.js

```diff
@@ -126,14 +121,14 @@ describe('<TablePagination />', () => {
         </table>,
       );
 
-      const backButton = wrapper.find(IconButton).at(0);
-      const nextButton = wrapper.find(IconButton).at(1);
-      expect(backButton.props().disabled).to.equal(true);
-      expect(nextButton.props().disabled).to.equal(false);
+      const backButton = getByLabelText('Previous page');
```

Comment:
We now [recommend `getByRole(role, { name: labelText })` for non-generic roles](https://testing-library.com/docs/guide-which-query#priority) such as `button`. Here we can apply 
```suggestion
      const backButton = getByRole('button', { name: 'Previous page' });
```